azure-spring-boot-samples/azure-active-directory-spring-boot-sample/pom.xml

```diff
@@ -33,6 +33,7 @@
         <dependency>
             <groupId>org.springframework.security.oauth</groupId>
             <artifactId>spring-security-oauth2</artifactId>
+            <version>2.2.0.RELEASE</version>
```

Comment:
That part has been replaced in Spring Security 5 and rather than hardcoding the previous jar, I'd prefer if you port the feature to what Spring Security 5 now offers.